thirdparty/pybind11/include/pybind11/pytypes.h

```diff
@@ -967,6 +1009,14 @@ class none : public object {
     none() : object(Py_None, borrowed_t{}) { }
 };
 
+#if PY_MAJOR_VERSION >= 3
```

Comment:
Do we plan to support Python 2 at all?

Reply:
I don't know if anything in our docs say we won't.

Reply:
I would think it over if we have the possibility to decide. 
Python 2 is already pretty old, not sure supporting that worth the effort.

Reply:
Are you asking because of the third party conditional?